src/main/java/seedu/address/model/project/ProjectPriority.java

```diff
@@ -0,0 +1,59 @@
+package seedu.address.model.project;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Project's priority in TaskHub.
+ * Guarantees: immutable; is valid as declared in {@link #isValidPriority(String)}
+ */
+public class ProjectPriority {
+    public static final String MESSAGE_CONSTRAINTS =
+            "Priority should only be low, normal, or high.";
+
+    public final String value;
+
```

Comment:
Since `priority` can only take 3 values, maybe enums will be more appropriate (and easier to debug)?

Reply:
Seems like @rayyan35p has added the enum. Will resolve this conversation.

Reply:
Did that! Thank you for the suggestion.